pkg/util/system/network_linux.go

```diff
@@ -0,0 +1,158 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package system
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/util/filesystem"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// NetworkRoute holds one network destination subnet and it's linked interface name
+type NetworkRoute struct {
+	Interface string
+	Subnet    uint64
+	Gateway   uint64
+	Mask      uint64
```

Comment:
Why are these 32-bit values stored in uint64?

Reply:
Easier to integrate with everything else as `ParseXXXX` always return `xxxx64`